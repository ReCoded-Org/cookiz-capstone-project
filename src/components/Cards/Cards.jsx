import Image from "next/image";
const Cards = () => {
    return (
        <div className='mt-20 ml-10 flex flex-col items-center justify-center md:ml-0'>
            {/* title part */}

            <div className='m-auto flex flex-col'>
                <div className=' flex  flex-col'>
                    <p className=' text-3xl font-bold text-black'>
                        Youâ€™ve missed home made food?
                    </p>
                    <p className=' pb-12  pt-2 text-xl   text-black'>
                        Pick your favorite chef, food or cousin.
                    </p>
                </div>

                <div className='mb-4  flex flex-row  flex-wrap justify-between gap-16'>
                    <div className='flex h-[350px]  w-[660px] flex-row items-center  justify-center rounded-[10px]  bg-[#9EE2B8] shadow-lg'>
                        <div className='basis-2/3 p-5'>
                            <div>
                                <Image
                                    alt='y'
                                    src='/images/cheff.png'
                                    height={50}
                                    width={50}
                                />
                            </div>
                            <div className='text-xl font-semibold text-black  sm:text-2xl lg:text-3xl'>
                                {" "}
                                Pick your chef order the food and get it when
                                you want
                            </div>
                            <div>
                                <Image
                                    alt='l'
                                    src='/images/soup.png'
                                    height={50}
                                    width={50}
                                />
                            </div>
                        </div>
                        <div className='flex items-center justify-center'>
                            <Image
                                alt='t'
                                src='/images/chef.png'
                                height={355}
                                width={225}
                                className='rounded-r-xl'
                            />
                        </div>
                    </div>

                    <div className='flex  h-[350px] w-[660px] flex-row rounded-[10px] bg-[#FFE7E3] shadow-lg '>
                        <div className='flex basis-2/3 flex-col justify-center gap-4 px-6 pt-12 '>
                            <p className='text-xl font-semibold text-black  sm:text-2xl lg:text-3xl'>
                                Get a 15% discount for your first order!
                            </p>
                            <button className='  box-border  h-10 w-32 bg-green-900 '>
                                <span className='h-5 w-20  text-sm font-bold leading-5 text-white sm:text-lg'>
                                    Order Now
                                </span>
                            </button>
                        </div>
                        <div>
                            <Image
                                src='/images/basket.png'
                                alt='x'
                                height={355}
                                width={225}
                                className='overflow-hidden rounded-r-lg '
                            />
                        </div>
                    </div>
                </div>
            </div>
        </div>
    );
};

export default Cards;
