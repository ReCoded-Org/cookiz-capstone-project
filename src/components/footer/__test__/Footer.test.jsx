import React from "react";
import renderer from "react-test-renderer";

import Cards from "../Footer";

// eslint-disable-next-line
it("renders correctly", () => {
    const tree = renderer.create(<Footer />).toJSON();
    // eslint-disable-next-line no-undef
    expect(tree).toMatchSnapshot();
});
