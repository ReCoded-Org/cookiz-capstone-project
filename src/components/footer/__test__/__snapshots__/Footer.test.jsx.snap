// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<footer
  className="bg-black text-white"
>
  <div
    className="grid-row-1 grid h-auto "
  >
    <div
      className="grid grid-cols-1 px-20 py-6 xl:grid-cols-5 "
    >
      <div
        className="col-span-2"
      >
        <div
          className="text-3xl font-extrabold hover:text-teal-200"
        >
          COOKIEZ
        </div>
        <div
          className="flex flex-row justify-start "
        >
          <span
            className="mr-4 mt-4 inline-flex cursor-pointer items-center rounded-full bg-gray-200 p-2 hover:bg-teal-500 hover:text-gray-100 sm:my-6"
          >
            <svg
              className="text-gray-600"
              fill="currentColor"
              height="1.75rem"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 1024 1024"
              width="1.75rem"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M880 112H144c-17.7 0-32 14.3-32 32v736c0 17.7 14.3 32 32 32h736c17.7 0 32-14.3 32-32V144c0-17.7-14.3-32-32-32zm-32 736H663.9V602.2h104l15.6-120.7H663.9v-77.1c0-35 9.7-58.8 59.8-58.8h63.9v-108c-11.1-1.5-49-4.8-93.2-4.8-92.2 0-155.3 56.3-155.3 159.6v89H434.9v120.7h104.3V848H176V176h672v672z"
              />
            </svg>
          </span>
          <span
            className="mr-4 mt-4 inline-flex cursor-pointer items-center rounded-full bg-gray-200 p-2 hover:bg-teal-500 hover:text-gray-100 sm:my-6"
          >
            <svg
              className="text-gray-600"
              fill="currentColor"
              height="1.75rem"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 1024 1024"
              width="1.75rem"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M928 254.3c-30.6 13.2-63.9 22.7-98.2 26.4a170.1 170.1 0 0 0 75-94 336.64 336.64 0 0 1-108.2 41.2A170.1 170.1 0 0 0 672 174c-94.5 0-170.5 76.6-170.5 170.6 0 13.2 1.6 26.4 4.2 39.1-141.5-7.4-267.7-75-351.6-178.5a169.32 169.32 0 0 0-23.2 86.1c0 59.2 30.1 111.4 76 142.1a172 172 0 0 1-77.1-21.7v2.1c0 82.9 58.6 151.6 136.7 167.4a180.6 180.6 0 0 1-44.9 5.8c-11.1 0-21.6-1.1-32.2-2.6C211 652 273.9 701.1 348.8 702.7c-58.6 45.9-132 72.9-211.7 72.9-14.3 0-27.5-.5-41.2-2.1C171.5 822 261.2 850 357.8 850 671.4 850 843 590.2 843 364.7c0-7.4 0-14.8-.5-22.2 33.2-24.3 62.3-54.4 85.5-88.2z"
              />
            </svg>
          </span>
          <span
            className="w-50 mr-4 mt-4 inline-flex cursor-pointer items-center rounded-full bg-gray-200 p-2 hover:bg-teal-500 hover:text-gray-100 sm:my-6"
          >
            <svg
              className="text-gray-600"
              fill="currentColor"
              height="1.75rem"
              stroke="currentColor"
              strokeWidth="0"
              style={
                Object {
                  "color": undefined,
                }
              }
              viewBox="0 0 1024 1024"
              width="1.75rem"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M512 306.9c-113.5 0-205.1 91.6-205.1 205.1S398.5 717.1 512 717.1 717.1 625.5 717.1 512 625.5 306.9 512 306.9zm0 338.4c-73.4 0-133.3-59.9-133.3-133.3S438.6 378.7 512 378.7 645.3 438.6 645.3 512 585.4 645.3 512 645.3zm213.5-394.6c-26.5 0-47.9 21.4-47.9 47.9s21.4 47.9 47.9 47.9 47.9-21.3 47.9-47.9a47.84 47.84 0 0 0-47.9-47.9zM911.8 512c0-55.2.5-109.9-2.6-165-3.1-64-17.7-120.8-64.5-167.6-46.9-46.9-103.6-61.4-167.6-64.5-55.2-3.1-109.9-2.6-165-2.6-55.2 0-109.9-.5-165 2.6-64 3.1-120.8 17.7-167.6 64.5C132.6 226.3 118.1 283 115 347c-3.1 55.2-2.6 109.9-2.6 165s-.5 109.9 2.6 165c3.1 64 17.7 120.8 64.5 167.6 46.9 46.9 103.6 61.4 167.6 64.5 55.2 3.1 109.9 2.6 165 2.6 55.2 0 109.9.5 165-2.6 64-3.1 120.8-17.7 167.6-64.5 46.9-46.9 61.4-103.6 64.5-167.6 3.2-55.1 2.6-109.8 2.6-165zm-88 235.8c-7.3 18.2-16.1 31.8-30.2 45.8-14.1 14.1-27.6 22.9-45.8 30.2C695.2 844.7 570.3 840 512 840c-58.3 0-183.3 4.7-235.9-16.1-18.2-7.3-31.8-16.1-45.8-30.2-14.1-14.1-22.9-27.6-30.2-45.8C179.3 695.2 184 570.3 184 512c0-58.3-4.7-183.3 16.1-235.9 7.3-18.2 16.1-31.8 30.2-45.8s27.6-22.9 45.8-30.2C328.7 179.3 453.7 184 512 184s183.3-4.7 235.9 16.1c18.2 7.3 31.8 16.1 45.8 30.2 14.1 14.1 22.9 27.6 30.2 45.8C844.7 328.7 840 453.7 840 512c0 58.3 4.7 183.2-16.2 235.8z"
              />
            </svg>
          </span>
        </div>
      </div>
      <div
        className="col-span-2"
      >
        <ul>
          <li
            className="my-4 hover:text-teal-300 "
          >
            Get help
          </li>
          <li
            className="my-4 hover:text-teal-300 "
          >
            Become a chef
          </li>
          <li
            className="my-4 hover:text-teal-300 "
          >
            Promotions
          </li>
          <li
            className="my-4 hover:text-teal-300 "
          >
            About Cookiez
          </li>
        </ul>
      </div>
      <ul>
        <li
          className="my-4 hover:text-teal-300 "
        >
          Chef near me
        </li>
        <li
          className="my-4 hover:text-teal-300 "
        >
          View all home made foods
        </li>
        <li
          className="my-4 hover:text-teal-300 "
        >
          View all cities
        </li>
        <li
          className="my-4 hover:text-teal-300 "
        >
          View all chefs
        </li>
      </ul>
    </div>
  </div>
</footer>
`;
